images/nginx-slim/build.sh

```diff
@@ -120,6 +128,11 @@ cd "$BUILD_PATH/nginx-$NGINX_VERSION"
 echo "Applying tls nginx patches..."
 patch -p1 < $BUILD_PATH/nginx__dynamic_tls_records.patch
 
+CC_OPT='-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64'
+if [[ ${ARCH} == "x86_64" ]]; then
```

Comment:
the option `-m64` cannot be located in the defaults. You need to move that option inside the if in the next line.